apps/web/src/modules/create-proposal/components/TransactionForm/Escrow/EscrowForm.tsx

```diff
@@ -187,23 +195,25 @@ const EscrowForm: React.FC<EscrowFormProps> = ({
                     'The wallet address to which funds will be released on milestone completions.'
                   }
                 />
-                <SmartInput
-                  type={TEXT}
-                  formik={formik}
-                  {...formik.getFieldProps('clientAddress')}
-                  id="clientAddress"
-                  inputLabel={'Client'}
-                  placeholder={'0x... or .eth'}
-                  isAddress={true}
-                  errorMessage={
-                    formik.touched.clientAddress && formik.errors.clientAddress
-                      ? formik.errors.clientAddress
-                      : undefined
-                  }
-                  helperText={
-                    'This is the wallet address that will control the escrow for releasing funds. This can be DAO or multisig of working group within the DAO.'
-                  }
-                />
+                {multiSig && (
```

Comment:
```suggestion
                {!multiSig && (
```
Shouldn't this be displayed if the multisig address is NOT found ? Or maybe we can always display it and disable the input if the multisig is found ?